Styling/README.md

```diff
@@ -204,7 +195,7 @@ A meta property expression can evaluate to any type. For example:
 }
```

Comment:
"can evaluate to any type." Is there a word missing after "type"?

Reply:
This is valid. If we wanted to be more specific we could say "data type", but I don't think that's necessary.